django/db/models/sql/compiler.py

```diff
@@ -957,7 +1007,7 @@ def execute_sql(self, result_type):
             if is_empty:
                 rows = aux_rows
                 is_empty = False
-        return rows
+        return (rows, returning_values)
```

Comment:
@shaib Looks like bulk_create's public API also does not need this. It does not even update IDs (any auto field) right now. May be we can add support for this later as a new feature?